banners/desktop/C24_WMDE_Desktop_DE_09/components/BannerVar.vue

```diff
@@ -1,164 +0,0 @@
-<template>
```

Comment:
Same reasoning here: If you leave this file alone, the next commit will show a small change (changing `<BannerText/>` and `<BannerSlides/>` to `<BannerTextVar/>` and `<BannerSlidesVar/>`) instead of being a completely new file. That will make it easier for the reviewer of the 3rd commit to see what changed.

Reply:
So, in the second commit  we make the `BannerVar.vue` be identical to `BannerCTtrl.vue` ?